NEWS.md

```diff
@@ -10,6 +10,7 @@
 * The `@code` field in the `qenv` class now holds `character`, not `expression`.
 * The `get_code` method  returns a single concatenated string of the code.
 * Added `within` support for `qenv.error` class.
+* Added `get_env,teal_data` method that allows to extract environment stored in `teal_data@env` slot.
```

Comment:
the term "teal_data" is unknown in the context of `teal.code` (it's in teal.data). Looking at the above entries we can skip class name to keep it simpler.

Reply:
a91bcc25bd936ff33f332403fab0cd7beb160a39